rules/validator/validator.go

```diff
@@ -171,20 +172,15 @@ func (v *validator) validateRollupRules(rrv map[string]*rules.RollupRuleView) er
 }
 
 func (v *validator) validateFilter(ruleName string, f string) (filters.TagFilterValueMap, error) {
-	filterValues, err := filters.ParseTagFilterValueMap(f)
+	filterValues, err := filters.ValidateTagsFilter(f)
```

Comment:
Do we need to wrap this error in the return statement below? Otherwise we'll lose the type information that it returns a validation error, no?

Reply:
Yeah so in the `wrapError` method, it'll wrap all errors that aren't validation errors or rule conflict errors as validation errors, so technically this will still be returned as an validation error.

That said, I can wrap this as a validation error explicitly if preferred.

Reply:
Using `wrapError` seems at the top level seems fine to me.